swiggyclone/accounts/signals.py

```diff
@@ -0,0 +1,11 @@
+rom django.contrib.auth.models import User
```

Comment:
did u test this code properly?

Reply:
Yes I have tested it. Its working fine.
https://lewiskori.com/blog/user-registration-and-authorization-on-a-django-api-with-djoser-and-json-web-tokens/